### PR TITLE
Remove ember-qunit-notifications and import qunit-notifications from NPM

### PR DESCRIPTION
**Pull request by trentmwillis**
Death to Bower!

This should be totally backwards compatible. Though, do we want to throw an error/give a warning after upgrading if folks still have `ember-qunit-notifications` in their `bower.json` as it is no longer used?

~~Should also wait to merge for a release of `qunit-notifications` with [this fix](https://github.com/DockYard/qunit-notifications/pull/26) in it.~~

cc @stefanpenner @bcardarella

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,15 +28,16 @@
   "dependencies": {
     "broccoli-babel-transpiler": "^5.5.0",
     "broccoli-concat": "^2.2.0",
+    "broccoli-funnel": "^1.0.1",
     "broccoli-merge-trees": "^1.1.0",
     "ember-cli-babel": "^5.1.5",
     "ember-cli-version-checker": "^1.1.4",
     "ember-qunit": "^0.4.18",
+    "qunit-notifications": "^0.1.0",
```
Comment (bcardarella) [inline package.json]: I just released 0.1.1 with the bower install removed

Comment (trentmwillis) [inline package.json]: @bcardarella thanks!

**Comment by trentmwillis** [comment]
@stefanpenner r?

**Comment by rwjblue** [comment]
Published as v2.0.3.

@trentmwillis - Can you update ember-cli's blueprint to match these changes?

**Comment by trentmwillis** [comment]
@rwjblue what updates would be needed? `ember-cli-qunit` is currently set to `^2.0.0` which seems fine unless we want to make `2.0.3` the minimum.

**Comment by rwjblue** [comment]
At least this https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/bower.json#L6

**Comment by trentmwillis** [comment]
Ah okay, can do, didn't look for that since it was part of the blueprint for this addon as well.

**Pull request merged**